README.md

```diff
@@ -20,57 +20,36 @@ All of this results in a more iterative process for developing manifests, one th
 
 > You should have Go installed *(minimum version 1.8.1)*. If not, follow the instructions in the [official installation guide](https://golang.org/doc/install#install).
 
-1. First ensure that you have a working `$GOPATH`:
-    ```
-    echo $GOPATH
-    ```
-    If your `$GOPATH` is empty, you'll need to permanently set it. On OSX, you can do this by adding the line `export GOPATH=$HOME/go` to the end of your `$HOME/.bash_profile`. (Otherwise [these instructions](https://github.com/golang/go/wiki/SettingGOPATH) may help).
-
-    To ensure that your Go binaries are runnable in any directory, run the following:
-    ```
-    PATH=$PATH:$GOPATH/bin
-    ```
-
-
-2. Now, to install ksonnet, copy and paste the following commands:
-
-    ```bash
-    # Download ksonnet
-    go get github.com/ksonnet/ksonnet
-
-    # Build and install binary under shortname `ks`
-    cd $GOPATH/src/github.com/ksonnet/ksonnet
-    make install
-    ```
-
-3. If your ksonnet is properly installed, you should be able to run the following `--help` command and see similar output:
+Copy and paste the following commands:
+```bash
+# Ensure a proper $GOPATH
```

Comment:
Can we move this piece down to the common issues section? That way we keep the install to ksonnet.